### ISSUE TITLE
wl-tab sets aria-checked attribute but has the role of tab which doesn't support it.

### ISSUE THREAD

**Issue by Lattyware** [andreasbm/weightless#77]
`wl-tab` sets [the `aria-checked` attribute](https://www.w3.org/TR/wai-aria-1.1/#aria-checked) when the tab is selected, but has the `tab` role, which does not support the attribute. Instead, [the `aria-selected` attribute](https://www.w3.org/TR/wai-aria-1.1/#aria-selected) should be used.
